airflow/models/taskinstance.py

```diff
@@ -365,7 +364,7 @@ class TaskInstance(Base, LoggingMixin):
     external_executor_id = Column(String(ID_LEN, **COLLATION_ARGS))
 
     # The trigger to resume on if we are in state DEFERRED
-    trigger_id = Column(BigInteger)
+    trigger_id = Column(Integer)
```

Comment:
All the DB doesn't like this and we are also not using BigInteger in the migration file. From mssql:
```
sqlalchemy.exc.ProgrammingError: (pyodbc.ProgrammingError) ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Column 'trigger.id' is not the same data type as referencing column 'task_instance.trigger_id' in foreign key 'task_instance_trigger_id_fkey'.
```